.swiftlint.yml

```diff
@@ -26,6 +26,7 @@ opt_in_rules:
   - unneeded_parentheses_in_closure_argument
   - extension_access_modifier
   - pattern_matching_keywords
+  - fallthrough
```

Comment:
This is unnecessary now that the rule is enabled by default.

Reply:
Ops, missed that. Will push a commit to `master` removing it.